Unable to build stand-alone GDASApp

Attempts to build GDASApp outside of g-w fail while adding gw-ci tests.

Below is the error message we get when building GDASApp `devleop` at 2be1fcd on Hera as `role.jedipara`.
```
-- ---------------------------------------------------------
-- Adding bundle project gdas
-- ---------------------------------------------------------
-- ^[[32m[GDASApp] (1.0.0) [2be1fcd]^[[m
-- Feature TESTS enabled
-- Found Python3: /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env-rocky8/install/intel/2021.5.0/python-3.10.13-2p2rady/bin/python3.10 (found version "3.10.13") found components: Interpreter
Traceback (most recent call last):
  File "/scratch1/NCEPDEV/da/role.jedipara/git/GDASApp/test/bundle/gdas/test/gw-ci/get_cycles.py", line 38, in <module>
    main()
  File "/scratch1/NCEPDEV/da/role.jedipara/git/GDASApp/test/bundle/gdas/test/gw-ci/get_cycles.py", line 32, in main
    idate_value = read_idate_from_yaml(args.yaml_file)
  File "/scratch1/NCEPDEV/da/role.jedipara/git/GDASApp/test/bundle/gdas/test/gw-ci/get_cycles.py", line 8, in read_idate_from_yaml
    with open(file_path, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/scratch1/NCEPDEV/da/role.jedipara/git/GDASApp/test/bundle/../../../ci/cases/pr/C48mx500_3DVarAOWCDA.yaml'
CMake Error at gdas/test/gw-ci/CMakeLists.txt:17 (string):
  string sub-command REPLACE requires at least four arguments.
Call Stack (most recent call first):
  gdas/test/gw-ci/CMakeLists.txt:63 (add_cycling_tests)

...


-- Configuring incomplete, errors occurred!
See also "/scratch1/NCEPDEV/da/role.jedipara/git/GDASApp/test/build/CMakeFiles/CMakeOutput.log".
See also "/scratch1/NCEPDEV/da/role.jedipara/git/GDASApp/test/build/CMakeFiles/CMakeError.log".
```

File `/scratch1/NCEPDEV/da/role.jedipara/git/GDASApp/test/bundle/../../../ci/cases/pr/C48mx500_3DVarAOWCDA.yaml` does not exisit because this is a stand-alone clone of GDASApp.

--- a/test/aero/CMakeLists.txt
+++ b/test/aero/CMakeLists.txt
@@ -1,10 +1,12 @@
 # tests that can run without needing all JEDI components
 
-# test to generate YAML for aerosol 3DVar
-add_test(NAME test_gdasapp_aero_gen_3dvar_yaml
-         COMMAND ${PROJECT_SOURCE_DIR}/test/aero/genyaml_3dvar.sh ${PROJECT_BINARY_DIR} ${PROJECT_SOURCE_DIR}
-         WORKING DIRECTORY ${PROJECT_BINARY_DIR}/test/testrun/)
-
+# test to generate YAML for aerosol 3DVar.
+if (WORKFLOW_TESTS)
+  add_test(NAME test_gdasapp_aero_gen_3dvar_yaml
+           COMMAND ${PROJECT_SOURCE_DIR}/test/aero/genyaml_3dvar.sh ${PROJECT_BINARY_DIR} ${PROJECT_SOURCE_DIR}
+           WORKING DIRECTORY ${PROJECT_BINARY_DIR}/test/testrun/)
+endif()
+       
 # tests that require the full build
 if(BUILD_GDASBUNDLE)
     # Test exgdas scripts from the global-worflow

--- a/test/gw-ci/CMakeLists.txt
+++ b/test/gw-ci/CMakeLists.txt
@@ -44,26 +44,26 @@ function(add_cycling_tests pslot YAML_PATH HOMEgfs RUNTESTS PROJECT_SOURCE_DIR T
   endforeach()
 endfunction()
 
-# Setup the environement
-set(HOMEgfs ${CMAKE_SOURCE_DIR}/../../..)
-set(RUNTESTS ${CMAKE_CURRENT_BINARY_DIR}/../../test/gw-ci)
-
-# 3DVAR WCDA, low-res
-# -------------------
-set(pslot "WCDA-3DVAR-C48mx500")
-set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C48mx500_3DVarAOWCDA.yaml)
-set(TASK_LIST
-  "gdasprepoceanobs"
-  "gdasmarinebmat"
-  "gdasmarineanlinit"
-  "gdasmarineanlvar"
-  "gdasmarineanlchkpt"
-  "gdasmarineanlfinal"
-  )
-add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
-
 option(RUN_GW_CI "Enable the global-workflow CI tests" OFF)
 if (RUN_GW_CI)
+  # Setup the environement
+  set(HOMEgfs ${CMAKE_SOURCE_DIR}/../../..)
+  set(RUNTESTS ${CMAKE_CURRENT_BINARY_DIR}/../../test/gw-ci)
+
+  # 3DVAR WCDA, low-res
+  # -------------------
+  set(pslot "WCDA-3DVAR-C48mx500")
+  set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C48mx500_3DVarAOWCDA.yaml)
+  set(TASK_LIST
+    "gdasprepoceanobs"
+    "gdasmarinebmat"
+    "gdasmarineanlinit"
+    "gdasmarineanlvar"
+    "gdasmarineanlchkpt"
+    "gdasmarineanlfinal"
+    )
+  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
+
   # Aero-Land DA, C96
   # -----------------
   set(pslot "Aero-Snow-3DVAR-C96")

--- a/test/gw-ci/CMakeLists.txt
+++ b/test/gw-ci/CMakeLists.txt
@@ -44,8 +44,7 @@ function(add_cycling_tests pslot YAML_PATH HOMEgfs RUNTESTS PROJECT_SOURCE_DIR T
   endforeach()
 endfunction()
 
-option(RUN_GW_CI "Enable the global-workflow CI tests" OFF)
-if (RUN_GW_CI)
+if (WORKFLOW_TESTS)
   # Setup the environement
   set(HOMEgfs ${CMAKE_SOURCE_DIR}/../../..)
   set(RUNTESTS ${CMAKE_CURRENT_BINARY_DIR}/../../test/gw-ci)
@@ -63,7 +62,10 @@ if (RUN_GW_CI)
     "gdasmarineanlfinal"
     )
   add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
+endif()
 
+option(RUN_GW_CI "Enable the global-workflow CI tests" OFF)
+if (RUN_GW_CI)
   # Aero-Land DA, C96
   # -----------------
   set(pslot "Aero-Snow-3DVAR-C96")

